CHANGES.txt

```diff
@@ -42,6 +42,13 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       registry query that returns a path that does not exist.  Multiple invocation
       paths were not prepared to handle the MissingConfiguration exception.  The
       MissingConfiguration exception type was removed.
+    - The detection of the msvc compiler executable (cl.exe) has been modified.  The
+      compiler detection no longer considers the host os environment path.  In addition,
+      existence of the msvc compiler executable is checked in the detection dictionary
+      and the scons ENV path before the detection dictionary is merged into the scons
+      ENV.  Different warnings are produced when the msvc compiler is not detected in the
```

Comment:
Probably worth indicating the old message and the new message here.  
I've searched CHANGES.txt before for message text to figure out which version changed it.

Reply:
The only *change* to the old message was inserting the word "requested".
new: "Could not find **requested** MSVC compiler 'cl'."
old: "Could not find MSVC compiler 'cl'."

Other than the word requested, the first two sentences are the same.

The third sentence is new (i.e., it did not appear before).

For all intents and purposes, the old message is *augmented* in certain circumstances (i.e., the compiler is not found in the requested dictionary and exists already in the environment path).

Is it really necessary?